README.md

```diff
@@ -68,60 +68,65 @@ With ESLint 2.x.x:
 }
 ```
 
-Finally, enable all of the rules that you would like to use.
+Finally, enable all of the rules that you would like to use.  Use [our preset](#user-content-recommended-configuration) to get reasonable defaults quickly, and/or choose your own:
 
-The plugin has a [recommended configuration](#user-content-recommended-configuration) that enforces React good practices.
+```json
+  "rules": {
+    "react/jsx-uses-react": "warn",
```

Comment:
Fixed.

Honestly, I wasn't sure what the right word was to use there.  `jsx-uses-react` fixes `eslint` errors, rather than introducing them.